ci/Jenkinsfile

```diff
@@ -70,6 +70,7 @@ pipeline {
     LD_LIBRARY_PATH = "${SQUISH_DIR}/lib:${SQUISH_DIR}/python3/lib:${LD_LIBRARY_PATH}"
 
     TESTRAIL_URL = 'https://ethstatus.testrail.net'
+    TESTRAIL_PROJECT_ID = 17
```

Comment:
any specific reason we keep the hardcoded value? Can we make a constant please?

Reply:
Could you please give me example, of how can I improve it?

Reply:
in my initial question i asked if we can keep a constant for this somewhere. if not then thats it @StateOf-Vlado